dev/schema-validate.js

```diff
@@ -32,7 +33,7 @@ class JsonSchemaAjv {
             strictTuples: false,
             allowUnionTypes: true
         });
-        const metaSchemaPath = require.resolve('ajv/dist/refs/json-schema-draft-07.json');
```

Comment:
This seems to have been broken after updating to modules, but I guess was never used directly. Should be fixed now with the equivalent resolution.